boards/itsybitsy_m4/README.md

```diff
@@ -48,3 +48,8 @@ cargo hf2 --release --example usb_serial --features="usb"
     Finished in 0.167s
 $
 ```
+
+If you are on Linux and hf2 fails to flash your board even if it is connected and in bootloader mode, you
+might need to add some `udev` rules if you have not done that yet.
+
+The example for adafruit boards is shown [here](https://crates.io/crates/hf2).
```

Comment:
It might be better to put this in the root README under section `## Getting code onto the devices with bootloaders: hf2-rs`

Reply:
Perhaps it can be on both places?

I added this on the BSPs READMEs because that's what you get to read when surfing crates.io

Reply:
Like I know it might be duplicated data, but should an user have to surf the project on git to find this info?

Reply:
ah dang good point. I see the benefit, I'm down for both places.